### ISSUE TITLE
More details of Inception model?

### ISSUE THREAD

**Issue by raingo** [tensorflow/tensorflow#402]
Please include the following details for the inception model included in the android and label-image example:

1. training dataset
2. testing dataset and what's the top-1 and top-5 performance
3. training parameters, number of iterations, learning rate, optimization algorithm

**Comment by vincentvanhoucke** [comment]
I believe that the currently released model corresponds to: http://arxiv.org/abs/1409.4842
@Yangqing would know exactly.
We don't have more details to release about that particular model beyond what's published.
A more recent version with lots of details about the training process is described here: http://arxiv.org/abs/1512.00567

**Comment by raingo** [comment]
Thanks for the information. Is it possible to release the inception-v3 model?
